public/js/lib/highlight/externals.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+import {escapeHtml}  from './utils'
+
+let externals = {}
+
+externals.checkLang = function (lang) {
```

Comment:
checkLang i.e. `lang` is not a good name for this, it implies that everything in there must be a language, which is a stretch for things lime mermaid. And might be completely wrong for plugins to come.

In the proposal in #20, I suggested "keyword" or "magic word" for that reason, but it might also be something completely different like "activator" (verbalised) or "type" (which is a bit too generic for my taste).

Also, the functionality is basically the same for all blocks that exist as of now. So it would be more fitting to make this an array (and have the `array#includes(keyword)` as part of the surrounding code, not for every individual highlighter.